modules/about-workbench-images.adoc

```diff
@@ -57,26 +53,27 @@ endif::[]
 ifdef::upstream[]
 | code-server 
 endif::[]
-a| With the code-server notebook image, you can customize your notebook environment to meet your needs using a variety of extensions to add new languages, themes, debuggers, and connect to additional services. Enhance the efficiency of your data science work with syntax highlighting, auto-indentation, and bracket matching, as well as an automatic task runner for seamless automation. See link:https://github.com/coder/code-server[code-server in GitHub] for more information. +
+a| With the code-server notebook image, which is an open source web version of VS Code, you can customize your notebook environment to meet your needs using a variety of extensions to add new languages, themes, debuggers, and connect to additional services. Enhance the efficiency of your data science work with syntax highlighting, auto-indentation, and bracket matching, as well as an automatic task runner for seamless automation. See link:https://github.com/coder/code-server[code-server in GitHub] for more information.
 
 NOTE: Elyra-based pipelines are not available with the code-server notebook image.
 
 ifdef::upstream[]
 | RStudio Server 
 a| Use the RStudio Server notebook image to access the RStudio IDE, an integrated development environment for R, a programming language for statistical computing and graphics.
-See link:https://posit.co/products/open-source/rstudio-server/[the RStudio Server site] for more information. +
```

Comment:
Just a check - does removing these '+' symbols affect the formatting of the text within the table? I think these were there to force new lines.